src/ScenarioPreview/ScenarioPreview.tsx

```diff
@@ -0,0 +1,167 @@
+      gap={1}
+      minWidth={0}
+      fontFamily="Montserrat"
+      data-testid="scenario-preview-wrapper"
+      sx={{ ...sx }}
+    >
+      <Box>
+        <Stack direction="row" spacing={1} minWidth={0}>
+          <img
+            src={image}
+            alt="scenario-image"
+            style={{
+              height: "44px",
+              marginBottom: "auto",
+              marginTop: "auto",
+              objectFit: "contain",
+              width: "78px"
+            }}
+          />
+          <Stack direction="column" minWidth={0}>
+            <Link
+              href={href}
+              target="_blank"
+              color="primary"
+              variant="subtitle2"
+              underline="hover"
+              textOverflow="ellipsis"
+              overflow="hidden"
+              data-testid="scenario-preview-name"
+              noWrap
+            >
+              {name}
+            </Link>
+
+            <Stack direction="row">
+              <Typography
+                variant="caption"
+                color="textPrimary"
+                data-testid="scenario-preview-description"
+                sx={{
+                  WebkitBoxOrient: "vertical",
+                  WebkitLineClamp: 2,
+                  display: "-webkit-box",
+                  overflow: "hidden",
+                  textOverflow: "ellipsis"
+                }}
+              >
+                {description}
+              </Typography>
+            </Stack>
+          </Stack>
+        </Stack>
+      </Box>
+      <Box gap={1}>
+        <Stack
+          direction={"row"}
+          gap={"12px"}
+          maxHeight={1}
+          justifyContent={"left"}
+          alignItems={"center"}
+        >
+          <Box data-testid="format-label" flex="0 1 auto">
+            <FormatLabel label={format} />
+          </Box>
+          <Box
+            data-testid="format-version-label"
+            flex="0 1 auto"
+            maxWidth={0.2}
+          >
+            <FormatVersionLabel label={formatVersion} />
+          </Box>
+          <Box
+            data-testid="file-label"
+            flex="0 1 auto"
+            maxWidth={"calc(40% - 12px)"}
+            alignItems="center"
+          >
+            <FileLabel label={file} />
+          </Box>
+          <Box
+            data-testid="road-label"
+            sx={{
+              display: "inline-block",
+              maxWidth: "calc(40% - 12px)",
+              overflow: "hidden",
+              textOverflow: "ellipsis",
+              whiteSpace: "nowrap"
+            }}
+          >
+            <RoadLabel label={roadName} href={roadHref} />
+          </Box>
+        </Stack>
+      </Box>
+      {hasOptionalProperty() && (
+        <>
+          <Divider />
+          <Box display="flex" flexDirection="column" gap={1}>
+            <Stack direction="row" gap={1}>
+              {createdAt && (
+                <Box data-testid="date-label" flex="0 1 auto">
```

Comment:
Suggestion to fix alignment issue
```suggestion
                <Box data-testid="date-label" flex="0 1 auto" alignItems="center">
```